pkg/backup/backup_compaction_test.go

```diff
@@ -340,6 +345,46 @@ crdb_internal.json_to_pb(
 			t, db, []string{"foo"}, "'nodelocal://1/backup/9'", start, end, opts,
 		)
 	})
+
+	t.Run("maybeStartCompactionJob", func(t *testing.T) {
+		db.Exec(t, "SET CLUSTER SETTING backup.compaction.threshold = 3")
```

Comment:
what test coverage does this test add over the e2e test below?

Reply:
I suppose it doesn't add any more coverage anymore, now that we have the code you recommended for forcing scheduled jobs to run.